util/disttask/idservice.go

```diff
@@ -26,3 +29,38 @@ func GenerateExecID(ip string, port uint) string {
 	portstring := fmt.Sprintf("%d", port)
 	return net.JoinHostPort(ip, portstring)
 }
+
+// MatchServerInfo will check if the schedulerID matched in all serverInfos
```

Comment:
Please add `.` at the end of a complete sentence.

Reply:
I will fix them all